system/settings/settings.xml

```diff
@@ -2249,7 +2253,7 @@
         </setting>
         <setting id="audiooutput.ac3transcode" type="boolean" parent="audiooutput.ac3passthrough" label="667" help="36429">
           <level>2</level>
-          <default>false</default>
+          <default>true</default>
```

Comment:
What's the intention behind it?

It can't overrule multichannel setting, it would transcode to ac3 when user has 2 channel layout and passthrough enabled though. Is this the intention?